### DISCUSSION TITLE
TS interface exact 1 space after colon

### DISCUSSION THREAD

**Discussion by Bentipa** [eslint/eslint#16956] [Q&A] [answered]
Hey,

when declaring an interface (or Type), eslint seems not to foce at least one spacing (whereas it does so for objects).
Valid:
`interface Test {
  att:string;
}
`
Invalid:
interface Test {
  att:  String; // Two Spaces
}

Is there a rule that covers this situation, enforcing exactly 1 space?
It seems like key-spacing is only looking at objects.

Thank you for any possible hint :)

**Comment by mdjermanovic** [comment, marked as answer]
Try with [`@typescript-eslint/key-spacing`](https://typescript-eslint.io/rules/key-spacing) rule. Here's a [typescript-eslint demo](https://typescript-eslint.io/play/#ts=4.9.3&sourceType=module&code=JYOwLgpgTgZghgYwgAgCoQM5mQbwLABQyycYYAXMQMphSgDmA3MgPQtoDuA9slQA6JMhAL5A&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQNaIE8BaZABwEMBjASwDsBzdKAewDNXIAacbSAAQAuBUikrRqpAURTw6AgPT5iZKnUYZIAd3LRakHgF8QBoA) with this rule.

**Comment by sam3k** [comment]
Hi @Bentipa, it is worth noting that TypeScript rules are maintained by a completely different team. See [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint). They might be able to assist you with more TypeScript related questions. 👍
